values.yaml

```diff
@@ -66,6 +66,10 @@ server:
   affinity: {}
   additionalVolumes: []
   additionalVolumeMounts: []
+  additionalEnvVars: []
+  additionalToolsEnvVars: []
+  additionalWebEnvVars: []
+  tls: []
```

Comment:
since this drops into the configmap - maybe move `tls` it to the config section?

Reply:
ok i will